static/css/addons-form.css

```diff
@@ -218,3 +218,8 @@
 #addon-config-settings .button-submit::-moz-focus-inner {
   border: 0;
 }
+
```

Comment:
nit: remove extra new line

Reply:
Not my code :-) Haven't touched that file.

Reply:
Oh wait, yes, I did add that extra space on purpose, to indicate that it's really a separate bit of code not related to the addons form, but more generally to addons overview. But there wasn't a CSS file for that, and this one seemed the closest match. 

I'll remove it.